python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -545,92 +553,83 @@ def _get_actions(self, data):
 
                 pc_data["lookup_hash"] = lookup_hash
                 pc_data["descriptor"] = pc_descriptor
-                contents_hash = pc_data["contents_hash"]
                 cached_data = []
+                logger.debug("Querying: %s", lookup_hash)
 
-                # We lock here because we cannot allow concurrent bootstraps to
-                # occur. We do it this early because we also want others to wait
-                # in the event that we end up caching what they need while they
-                # wait for the lock to be released.
-                with self._LOCK:
-                    logger.debug("Querying: %s", lookup_hash)
-
-                    try:
-                        cursor.execute(
-                            "SELECT commands, contents_hash FROM engine_commands WHERE lookup_hash=?",
-                            (lookup_hash,)
-                        )
-                        cached_data = list(cursor.fetchone() or [])
-                    except sqlite3.OperationalError:
-                        # This means the sqlite database hasn't been setup at all.
-                        # In that case, we just continue on with cached_data set
-                        # to an empty list, which will cause the caching subprocess
-                        # to be spawned, which will setup the database and populate
-                        # with the data we need. The behavior as a result of this
-                        # is the same as if we ended up with a cache miss or an
-                        # invalidated result due to a contents_hash mismatch.
-                        logger.debug(
-                            "Cache query failed due to missing table. "
-                            "Triggering caching subprocess..."
+                try:
```

Comment:
NOTE: This block of change is mostly just due to the tabbing changing after removing the lock context.